style.css

```diff
@@ -0,0 +1,262 @@
+html {
+  margin: 0;
+  padding: 0;
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 0.01px;
+  margin-inline-start: 0.01px;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+  background: #fff;
+}
+
+li {
+  text-decoration: none;
+  list-style: none;
+  font-family: "Poppins", sans-serif;
+  color: #092c69;
+  font-weight: 530;
+  font-size: 12px;
+  font-style: normal;
+}
+
+ul {
+  list-style-type: none;
+  margin-block-end: 0.01px;
+  padding-inline-start: 0.01px;
+}
+
+h2 {
+  font-family: "Poppins", sans-serif;
+  color: #172b4d;
+  font-size: 33px;
+  font-weight: bolder;
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 0.01px;
+  margin-inline-start: 0.01px;
+}
+
+p {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  text-align: left;
+  font-size: 16px;
+  line-height: 25px;
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 0.01px;
+  margin-inline-start: 0.01px;
+}
+
+button {
+  font-size: 17px;
+  color: #396df2;
+  background: #fff;
+  border: none;
+  font-weight: 550;
+}
+
```

Comment:
- [ ] Kindly add button interaction styles as required by the previous reviewer. On hover, the background color of the button should change based on the design in Figma and as well as on active state.

See the different states of the button from the Figma design
![image](https://user-images.githubusercontent.com/50941074/159135042-a4d912aa-0c7b-49f9-b791-4ab834bff39b.png)

Hint: 
```code
button:hover {
  background-color: #6070FF;
  color: #fff;
}

button:focus {
  background-color: #2230D2;
  color: #fff;
}